_topic_map.yml

```diff
@@ -348,7 +348,7 @@ Topics:
     File: nodes-scheduler-taints-tolerations
   - Name: Constraining pod placement using node selectors
     File: nodes-scheduler-node-selectors
-  - Name: Keeping your cluster balanced using the descheduler
+  - Name: Keeping your cluster balanced using the Descheduler Operator
```

Comment:
@mburke5678 - Descheduler is still reponsible for balancing the cluster, the management of descheduler now happen through descheduler operator.